README.md

```diff
@@ -7,84 +7,24 @@
 
 ABLincoln is a PHP-based toolkit for online field experimentation, ported from
 Facebook's [PlanOut][] software and released as an easily extendable
-[Composer][] package. ABLincoln makes it easy to deploy and maintain
-sophisticated experiments and to quickly iterate on these experiments, while
-satisfying the constraints of large-scale Internet services with many users.
+[Composer][installation] package. ABLincoln makes it easy to deploy and
```

Comment:
How about: ABLincoln is a PHP-based implementation of Facebook's PlanOut, a framework for online field experimentation. ?